### ISSUE TITLE
[All] Change to UUID from Increment for ids in all databases

### ISSUE THREAD

**Issue by emilwihlander** [Dsek-LTH/member-page#208]
## Short description of the feature
It's nicer to have UUID.

Do this issue depend on another issue? Write Depends on #Issue_number here

[ ] Relates to frontend
[X] Relates to the backend
[X] Requires modification of GraphQL
[X] Requires modification of Database
[ ] Requires creating a new service
